packages/grafana-ui/src/components/Combobox/Combobox.story.tsx

```diff
@@ -130,6 +130,39 @@ export const CustomValue: StoryObj<PropsAndCustomArgs> = {
   },
 };
 
+const AsyncStory: StoryFn<PropsAndCustomArgs> = (args) => {
+  const [value, setValue] = useState(args.value);
+  return (
+    <Field label="Test input" description="Input with a few options">
+      <Combobox
+        id="test-combobox"
+        options={async (inputValue) => {
```

Comment:
Should we be actually using the `inputValue` to search?

Reply:
What do you mean?

Reply:
Yep that's what I meant, I was just a bit confused when I was typing and nothing was being filtered 😅  Just a simple `.filter` would be enough I think.